eo-runtime/src/main/java/org/eolang/BytesOf.java

```diff
@@ -208,4 +236,34 @@ public boolean equals(final Object other) {
     public int hashCode() {
         return Arrays.hashCode(this.data);
     }
+
+    /**
+     * Count leading zero bits.
+     * @param num Byte.
+     * @return Number between 0 and 8.
+     */
+    static byte numberOfLeadingZeros(final byte num) {
```

Comment:
@andreoss can this be private? There is no reason to have this package-private other than tests. Exposing members solely to test them is a design flaw.

Reply:
@mximp Done